src/main.c

```diff
@@ -620,18 +609,14 @@ static void library_main_helper(struct libargs_s *args) {
             }
             break;
         case GET_PRINTABLE_AMOUNT:
-            // ensure result is zero if an exception is thrown (compatibility breaking, disabled
-            // until LL is ready)
-            // args->get_printable_amount->result = 0;
-            // args->get_printable_amount->result =
             handle_get_printable_amount(args->get_printable_amount, args->coin_config);
             break;
         default:
             break;
```

Comment:
@fbeutin-ledger  Don't you need to call `finalize_exchange_sign_transaction` here to ensure exchange gets the correct return value ?

Reply:
finalize_exchange_sign_transaction is only for `SIGN_TRANSACTION` flow
The `handle_get_printable_amount` will simply return to caller (`library_main()`) that will call `os_lib_end`